cmd/dep/init.go

```diff
@@ -73,7 +73,7 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 		return err
 	}
 	if mok {
-		return errors.Errorf("manifest file %q already exists", mf)
+		return errors.New("manifest file already exists")
```

Comment:
This isn't the best - we don't want to remove useful information from an error message that's shown to humans just to make the tests pass.

Let's change it to `return errors.Errorf("manifest already exists: %s", mf)`, then change the check from exact equality to `strings.Contains()`.